Source/Orts.Simulation/packages.config

```diff
@@ -1,4 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="Microsoft.CodeAnalysis.Analyzers" version="3.3.2" targetFramework="net472" developmentDependency="true" />
+  <package id="Microsoft.CodeAnalysis.Common" version="4.0.1" targetFramework="net472" />
+  <package id="Microsoft.CodeAnalysis.CSharp" version="4.0.1" targetFramework="net472" />
   <package id="Microsoft.CodeDom.Providers.DotNetCompilerPlatform" version="3.6.0" targetFramework="net472" />
```

Comment:
Please remove the old package

Reply:
Ah, you are absolutely right. I'm travelling on this weekend though, so probably won't be able to do it for some time, sorry for that. (Feel  free to push a commit, if this is the barrier for merging.)